platforms/nuttx/src/px4/common/tasks.cpp

```diff
@@ -100,7 +101,15 @@ const char *px4_get_taskname(void)
 int px4_exec(const char *appname, char *const *argv, const char *redirfile, int oflags)
 {
 #ifdef CONFIG_BUILTIN
-	return exec_builtin(appname, argv, NULL, redirfile, oflags);
+	FAR const struct nsh_param_s param = {
```

Comment:
FAR not needed here as this is not a pointer

Reply:
copy-paste coding